### PR TITLE
UI modification on index, show and new page

### PR DESCRIPTION
**Pull request by HaominZhang**

### CODE REVIEW

**Comment by bboe** [comment]
:+1: Looks great!

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-      <a class="nav-link" href="#">Tags</a>
-    </li>
-    <li class="nav-item">
-    </li>
-  </ul>
+<nav class="navbar navbar-light navbar-fixed-top bg-faded", style="background-color: #e3f2fd;">
+  <div class="container">
+    <%= link_to 'ImageUploader', root_path, class: 'navbar-brand'%>
+      <ul class="nav navbar-nav">
+        <li class="nav-item active">
```
Comment (rmacklin) [inline app/views/layouts/_navigation.html.erb]: This hardcodes the first link to always be active. Aaron's suggestion was to remove the `active` class for now and as an exercise try to dynamically add the `active` class to the appropriate link.

File: app/views/pictures/index.html.erb [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
 
-</table>
+<div id="pictures" class="container">
+  <div class="row">
+    <div class="col-sm-12">
+      <%=link_to "New Image", new_picture_path, class: 'btn btn-warning' %>
+    </div>
+  </div>
+  <br>
+  <% @pictures.each_slice(3) do |slice| %>
```
Comment (rmacklin) [inline app/views/pictures/index.html.erb]: You may want to go over this with @aaronpanch. It shouldn't be necessary to use `each_slice` and create multiple `.row`s.

**Comment by rmacklin** [comment]
@HaominZhang I noticed that your staging application is down (and I suspect that it has been down at least since #10 was merged). Have you been checking your deployed apps after you merge PRs? It's a good idea to do that after each merge because sometimes things will work in the local development environment but fail in production.

**Pull request merged**